package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This file was probably generated during npm calls. It seems this should not be commited?!